src/sum.js

```diff
@@ -0,0 +1,15 @@
+export default sum
+
+/**
+* Original Source: http://stackoverflow.com/questions/1230233/how-to-find-the-sum-of-an-array-of-numbers
+*
+* This method will sum up all the array elements, assuming the elements are all numbers
+*
+* @param {Array} arr - the arr to be summed up
+*
+*/
+
+function sum(arr) {
+  return arr.reduce(function(a, b) {
```

Comment:
Oh yes! I only committed it. I just pushed it to `origin pr/sum`. Looks like everything passed. Is the next step for me to wait for you to accept it? :)